### PR TITLE
feat: table of contents in README

### PR DESCRIPTION
**Pull request by Chriscbr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,34 @@ and that meet our community's quality bar.
 One of the cool things about trusted libraries is that we take care of building, testing and
 publishing them for you.
 
+<!-- WINGLIBS_TOC_START -->
+
+## Table of contents
```
Comment (Chriscbr) [inline README.md:13]: Clever header suggestions welcome ("the humble winglibs catalog" etc.)

Comment (eladb) [reply README.md:13]: "All the libs"?

**Review by eladb — APPROVED** [review]
Sweet!

File: README.md (hunk 1 above)
Comment (Chriscbr) [reply README.md:13]: For political reasons I'll pass on this one 😆

File: readme.w [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    let start = lines.indexOf("<!-- WINGLIBS_TOC_START -->");
+    let end = lines.indexOf("<!-- WINGLIBS_TOC_END -->");
+
+    if (start == -1 || end == -1) {
+      throw "Could not find WINGLIBS_TOC_START or WINGLIBS_TOC_END in README.md";
+    }
+
+    let newLines = Util.arraySlice(lines, 0, start + 1);
+    newLines.push("");
+    newLines.push("## Table of contents");
```
Comment (MarkMcCulloh) [inline readme.w:20]: I'm usually a big fan of headers but I think this would look better without one

**Pull request merged**